VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -86,6 +86,12 @@ export abstract class WaitingRoomBaseDirective {
     displayStartPrivateConsultationModal: boolean;
     displayJoinPrivateConsultationModal: boolean;
 
+    panelTypes = ['Participants', 'Chat'];
+    panelStates = {
+        Participants: true,
+        Chat: false
+    };
```

Comment:
I realise this is from before but this seems like an odd way to set this up. It would be better to have 1. Either an array of objects or maybe use Object.keys()

Reply:
left as it needs refactor